app/renderer/components/tabs/content/tabIcon.js

```diff
@@ -43,14 +30,16 @@ class TabIcon extends ImmutableComponent {
       onDragStart={this.props.onDragStart}
       draggable={this.props.draggable}
       onClick={this.props.onClick}
+      style={this.props.style}
       {...altProps}
     >
       {
         this.props.symbol
           ? <span
             className={cx({
               [this.props.symbol]: true,
-              [css(styles.tabIcon)]: true
+              [css(styles.tabIcon)]: true,
+              [css(styles.tabIcon_hasSymbol)]: this.props.symbolContent
```

Comment:
Would you mind combining these two lines into `[css(styles.tabIcon, this.props.symbolContent && styles.tabIcon_hasSymbol)]: true`, since `cx` basically does what this document describes: https://github.com/Khan/aphrodite#dont-do-this?

> Why does it matter? Although the second one will produce a valid class name, it cannot guarantee that the bar styles will override the foo ones.

Reply:
Thanks for the spot - I had just read that "don't do this" section today and had the same thought